app/scripts/templates/sign_in.mustache

```diff
@@ -66,7 +66,7 @@
       {{^suggestedAccount}}
         <form novalidate>
         <div class="input-row">
-          <input type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" {{^email}}autofocus{{/email}}>
+          <input type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" {{^email}}autofocus{{/email}}>
```

Comment:
The reset_password template also has an email field. This should probably be added there too.